elastic/indexer/insta_users/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.13 as builder
+WORKDIR /app
+COPY go.mod mod.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -o exe elastic/indexer/insta_users/main.go
```

Comment:
let's call this `es-users-indexer` to stay in line with the other binary names - the name is seen when you look at the process list, `exe` ist a bit missleading when you look at `top` or `htop` inside the node/pod